Error while checking build/O2/fullCI for 9f9f0f36527149bc7c877411b058fc430bac57ab at 2024-01-31 15:10:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/O2-full-system-test-latest/log
./tofReco.log[572527:tof-digit-reader]: [14:02:41][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./trdTrkltTransf.log[545775:TRDTrackletReader]: [13:52:52][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./itsreco.log[495360:its-digit-reader]: [13:34:05][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./hmpidMatchTracks.log[589479:HMP-ClusterReader]: [14:09:03][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./hmpidMatchTracks.log[589485:internal-dpl-ccdb-backend]: Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3014] Unable to open file  /eos/aliceupb/grid/14/49841/3f3c7626-99d9-11ee-b9a2-08f1eaf0250c; Network is unreachable
./hmpidMatchTracks.log[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Failed to open file alien:///alice/data/CCDB/GLO/Param/MatLUT/06/29739/3f0d0f81-99d9-11ee-b9a2-200114580202?filetype=raw
./hmpidMatchTracks.log[589485:internal-dpl-ccdb-backend]: [14:10:10][FATAL] Unable to find object GLO/Param/MatLUT/1546300800000
./hmpidMatchTracks.log[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Exception caught: Fatal error 
./hmpidMatchTracks.log[589485:internal-dpl-ccdb-backend]: [14:10:11][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-hmpid-matcher-workflow, device shutting down. Reason: Fatal error
./hmpidMatchTracks.log[ERROR] Workflow crashed - PID 589485 (internal-dpl-ccdb-backend) did not exit correctly however it's not clear why. Exit code forced to 128.
./itstpcMatch.log[532178:ft0-recpoints-reader]: [13:47:55][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./trdMatch.log[551249:TRDTrackletReader]: [13:55:08][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./fv0reco.log[523173:fv0-digit-reader]: [13:44:26][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./tofMatchTracks.log[578397:itstpc-track-reader]: [14:05:07][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./tpcreco.log[475449:tpc-digit-reader]: [13:26:24][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./tpcreco.log[475453:tpc-tracker]: [13:33:59][ERROR] Expected Lifetime::Timeframe data TPC/TRIGGERWORDS/0 was not created for timeslice 0 and might result in dropped timeframes
./tpcreco.log[475456:tpc-track-writer]: [13:33:59][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./tpcreco.log[475455:tpc-native-cluster-writer]: [13:33:59][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./mftmchMatch.log[564289:mch-tracks-reader0]: [13:59:34][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./ft0reco.log[518142:ft0-digit-reader]: [13:42:27][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./mchreco.log[511566:mch-digit-reader]: [13:40:17][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./mchreco.log[511577:mch-error-writer]: [13:42:22][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./mchreco.log[511578:mch-track-writer]: [13:42:22][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./sim-challenge.logDetected critical problem in logfile hmpidMatchTracks.log
./sim-challenge.loghmpidMatchTracks.log-[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Failed to open file alien:///alice/data/CCDB/GLO/Param/MatLUT/06/29739/3f0d0f81-99d9-11ee-b9a2-200114580202?filetype=raw
./sim-challenge.loghmpidMatchTracks.log-[589485:internal-dpl-ccdb-backend]: [14:10:10][FATAL] Unable to find object GLO/Param/MatLUT/1546300800000
./sim-challenge.loghmpidMatchTracks.log:[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Exception caught: Fatal error 
./sim-challenge.log[589479:HMP-ClusterReader]: [14:09:03][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./sim-challenge.log[589485:internal-dpl-ccdb-backend]: Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3014] Unable to open file  /eos/aliceupb/grid/14/49841/3f3c7626-99d9-11ee-b9a2-08f1eaf0250c; Network is unreachable
./sim-challenge.log[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Failed to open file alien:///alice/data/CCDB/GLO/Param/MatLUT/06/29739/3f0d0f81-99d9-11ee-b9a2-200114580202?filetype=raw
./sim-challenge.log[589485:internal-dpl-ccdb-backend]: [14:10:10][FATAL] Unable to find object GLO/Param/MatLUT/1546300800000
./sim-challenge.log[589485:internal-dpl-ccdb-backend]: [14:10:10][ERROR] Exception caught: Fatal error 
./sim-challenge.log[589485:internal-dpl-ccdb-backend]: [14:10:11][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-hmpid-matcher-workflow, device shutting down. Reason: Fatal error
./sim-challenge.log[ERROR] Workflow crashed - PID 589485 (internal-dpl-ccdb-backend) did not exit correctly however it's not clear why. Exit code forced to 128.
./mftreco.log[504365:mft-digit-reader]: [13:37:29][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./midreco.log[526663:MIDDigitsReader]: [13:45:51][ERROR] Expected Lifetime::Timeframe data FLP/DISTSUBTIMEFRAME/52443 was not created for timeslice 0 and might result in dropped timeframes
./midreco.log[526668:MIDRecoWriter]: [13:47:41][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./mchmidMatch.log[531834:muon-track-writer]: [13:47:49][ERROR] Some Lifetime::Timeframe data got dropped starting at 1
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/37}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/38}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/40}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/42}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/43}
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12609/9f9f0f36527149bc7c877411b058fc430bac57ab/build_O2_fullCI/pretty.html).

--- a/prodtests/full-system-test/dpl-workflow.sh
+++ b/prodtests/full-system-test/dpl-workflow.sh
@@ -293,6 +293,13 @@ elif [[ -z "$DISABLE_ROOT_OUTPUT" ]] && has_detector_reco TOF && ! has_detector_
   TOF_OUTPUT+=",digits"
 fi
 
+# adding FIT info to TOF matching for calib (only if FIT is present)
+if  has_detector_reco FT0 ; then
+: ${TOF_MATCH_OPT="--use-fit"}
+else
+: ${TOF_MATCH_OPT=}
+fi
+
 if has_detector_calib PHS && workflow_has_parameter CALIB; then
   PHS_CONFIG+=" --fullclu-output"
 fi
@@ -478,7 +485,7 @@ has_detector_reco FT0 && ! has_detector_from_global_reader FT0 && add_W o2-ft0-r
 has_detector_reco TRD && ! has_detector_from_global_reader TRD && add_W o2-trd-tracklet-transformer "--disable-irframe-reader $DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_FILTER_CONFIG --pipeline $(get_N TRDTRACKLETTRANSFORMER TRD REST 1 TRDTRKTRANS)"
 has_detectors_reco ITS TPC && has_detector_matching ITSTPC && add_W o2-tpcits-match-workflow "$DISABLE_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $SEND_ITSTPC_DTGL  $TPC_CORR_OPT --nthreads $ITSTPC_THREADS --pipeline $(get_N itstpc-track-matcher MATCH REST $ITSTPC_THREADS TPCITS)" "$ITSTPC_CONFIG_KEY;$INTERACTION_TAG_CONFIG_KEY;$ITSMFT_STROBES;$ITSEXTRAERR;$TPC_CORR_KEY"
 has_detector_reco TRD && [[ ! -z "$TRD_SOURCES" ]] && add_W o2-trd-global-tracking "$DISABLE_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_CONFIG $TRD_FILTER_CONFIG $TPC_CORR_OPT --track-sources $TRD_SOURCES --pipeline $(get_N trd-globaltracking_TPC_ITS-TPC_ TRD REST 1 TRDTRK),$(get_N trd-globaltracking_TPC_FT0_ITS-TPC_ TRD REST 1 TRDTRK),$(get_N trd-globaltracking_TPC_FT0_ITS-TPC_CTP_ TRD REST 1 TRDTRK)" "$TRD_CONFIG_KEY;$INTERACTION_TAG_CONFIG_KEY;$ITSMFT_STROBES;$ITSEXTRAERR;$TPC_CORR_KEY"
-has_detector_reco TOF && [[ ! -z "$TOF_SOURCES" ]] && add_W o2-tof-matcher-workflow "$DISABLE_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TPC_CORR_OPT ${TOFMATCH_THREADS:+--tof-lanes ${TOFMATCH_THREADS}} --track-sources $TOF_SOURCES --pipeline $(get_N tof-matcher TOF REST 1 TOFMATCH)" "$ITSMFT_STROBES;$ITSEXTRAERR;$TPC_CORR_KEY"
+has_detector_reco TOF && [[ ! -z "$TOF_SOURCES" ]] && add_W o2-tof-matcher-workflow "$TOF_MATCH_OPT $DISABLE_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TPC_CORR_OPT ${TOFMATCH_THREADS:+--tof-lanes ${TOFMATCH_THREADS}} --track-sources $TOF_SOURCES --pipeline $(get_N tof-matcher TOF REST 1 TOFMATCH)" "$ITSMFT_STROBES;$ITSEXTRAERR;$TPC_CORR_KEY"
 has_detectors TPC && [[ -z "$DISABLE_ROOT_OUTPUT" && "${SKIP_TPC_CLUSTERSTRACKS_OUTPUT:-}" != 1 ]] && ! has_detector_from_global_reader TPC && add_W o2-tpc-reco-workflow "--input-type pass-through --output-type clusters,tpc-triggers,tracks,send-clusters-per-sector $DISABLE_MC"
 
 # ---------------------------------------------------------------------------------------------------------------------

